src/main/java/org/broadinstitute/hellbender/engine/ReadShard.java

```diff
@@ -147,9 +160,17 @@ public boolean containsStartPosition( final Locatable loc ) {
      */
     @Override
```

Comment:
enhance the documentation here to state that the filters will be applied before the downsamping